src/app/layouts/card/card-styling-sample/card-styling-sample.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="sample-column card-wrapper">
```

Comment:
We need a **config instance** for this sample in the card's config generator in order to run it in stackblitz.

Reply:
Done

Reply:
fixed